development/TESTING.md

```diff
@@ -21,4 +21,4 @@ mvn test
 
 **Note**: These tests are currently deprecated.
 
-Please refer to the [integration tests README](integration-tests/README.md) for more information.
```

Comment:
This should be `../integration-tests/README.md` 
A relative pat should be used, this way we would link to the doc in the same branch that we are using. Otherwise we would always be redirected to the main version of the doc